kbase-extension/scss/partials/_jobLog.scss

```diff
@@ -23,61 +25,99 @@
   }
 }
 
-.kblog {
+.kblog,
```

Comment:
Adding styles for the BEM-ified job logs. I left the old styles in place for now as I haven't yet finished investigating to check there are no other places where logs are generated/displayed.